lightning/restore/tidb.go

```diff
@@ -137,52 +137,69 @@ func InitSchema(ctx context.Context, g glue.Glue, database string, tablesSchema
 	}
 
 	task := logger.Begin(zap.InfoLevel, "create tables")
+	var sqlCreateStmts []string
 	for tbl, sqlCreateTable := range tablesSchema {
 		task.Debug("create table", zap.String("schema", sqlCreateTable))
 
-		sqlCreateTable, err = createTableIfNotExistsStmt(g.GetParser(), sqlCreateTable, database, tbl)
+		sqlCreateStmts, err = createTableIfNotExistsStmt(g.GetParser(), sqlCreateTable, database, tbl)
 		if err != nil {
 			break
 		}
 
-		err = sqlExecutor.ExecuteWithLog(
-			ctx,
-			sqlCreateTable,
-			"create table",
-			logger.With(zap.String("table", common.UniqueTable(database, tbl))),
-		)
-		if err != nil {
-			break
+		//TODO: maybe we should put these createStems into a transaction
```

Comment:
not sure we could do this. seems DDL will do "an implicit commit after executing"

https://dev.mysql.com/doc/refman/5.6/en/implicit-commit.html

Reply:
These actions are not all DDLs. Some of them are session variables setting.